### ISSUE TITLE
Add UIA support for tooltips

### ISSUE THREAD

**Issue by mwcampbell** [AccessKit/accesskit#30]
UIA has specific events that should be raised when tooltips are shown or hidden. Make sure our implementation of these events works as expected with all Windows screen readers.
